openedx/license-manager
# ENT-2834: Adding celery tasks to send email asynchronously

## Description

* Celery settings and constraints cleanup.
* Celery email tasks implemented with tests.

Link to the associated ticket: https://openedx.atlassian.net/browse/ENT-2834

## Testing Instructions

- Seed subscription data in your devstack environment
- Visit http://localhost:18170/api/v1/subscriptions/UUID/licenses/remind/ for an existing SubscriptionPlan
- Enter appropriate data in the POST form and submit
- Run 'make worker-logs' and observe the email sent printed to the console 

## Post-review

Squash commits into discrete sets of changes
